lib/minicron/hub/controllers/jobs.rb

```diff
@@ -459,4 +459,87 @@ class Minicron::Hub::App
       erb :'jobs/schedules/delete', :layout => :'layouts/app'
     end
   end
+
+  get '/jobs/import/:host' do
```

Comment:
Could this be moved to the `hosts` controller and have the route as `/hosts/:id/import` instead? I think this more closely follows how the other routes work.